spec/controllers/openid_connect/authorization_controller_spec.rb

```diff
@@ -107,6 +114,7 @@
 
         context 'with ial1 requested using acr_values' do
           it 'tracks IAL1 authentication event' do
+            travel_to Time.zone.now + 15.seconds
```

Comment:
to make the time easier to track, I would consider using a `let` block to freeze a specific time

so like

```ruby
let(:now) { Time.zone.now)

around do |ex|
  freeze_time(now) { ex.run }
end

...

it 'tracks events' do
  travel_to now + 15.seconds
  ...

```
```

Reply:
I see. I have added that as a let block and updated those tracked events. Thanks!